docs/src/atomistic/engines/ipi.rst

```diff
@@ -0,0 +1,69 @@
+.. _engine-ipi:
+
+i-PI
+====
+
+.. list-table::
+   :header-rows: 1
+
+   * - Official website
+     - How is metatensor supported?
+   * - https://ipi-code.org/
+     - In the official version
+
+
+Supported model outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Only the :ref:`energy <energy-output>` output is supported, and used to run path
+integral simulations, incorporating quantum nuclear effects in the statistical
+sampling.
+
+How to install the code
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The metatensor interface is part of i-PI since version 3.0. Please refer to
+`i-PI documentation`_ about how to install it.
+
+.. _i-PI documentation: https://ipi-code.org/i-pi/getting-started.html#installing-i-pi
+
+How to use the code
+^^^^^^^^^^^^^^^^^^^
+
+The metatensor interface in i-PI provides a custom i-PI client that can be used
+in combination with an i-PI server to run simulations. This client is managed
+with the ``i-pi-driver-py`` command.
+
+.. code-block:: bash
+
+    # minimal version
+    i-pi-driver-py -m metatensor -o template.xyz,model.pt
+
+    # all possible options
+    i-pi-driver-py -m metatensor -o template.xyz,model.pt,device=cpu,extensions=path/to/extensions,check_consistency=False
+
+The minimal options to give to the ``metatensor`` client are the path to a
+template structure for the simulated system (``template.xyz`` in the example
+above) and the path to the metatensor model (``model.pt`` above). The template
```

Comment:
maybe we should link to a tutorial that creates an actual model tat people can download.